pkg/tuned/Makefile.am

```diff
@@ -1,26 +1,30 @@
+TUNED_PKGDIR=pkg/tuned
+
 tuneddir = $(pkgdatadir)/tuned
 nodist_tuned_DATA = \
-	pkg/tuned/bundle.min.js.gz \
+	$(TUNED_PKGDIR)/bundle.min.js.gz \
```

Comment:
If it's easy, could you include refactoring/renaming in a separate commit? That'll make this easier to review.